src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -43,6 +43,11 @@
      * Subdirectory where the report will be created.
      */
     public static final String BASE_DIRECTORY = "cucumber-html-reports";
+    /**
+     * Separator between main directory name and specified suffix
+     */
+    public static final String SUFFIX_SEPARATOR = "_";
```

Comment:
reduntant with https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/java/net/masterthought/cucumber/Configuration.java#L23

Reply:
I must have missed during merges.  Variable PREFIX_SEPARATOR in Configuration class is no longer used - and I pushed another commit to remove it.

Reply:
I must have missed something during merges.  Variable PREFIX_SEPARATOR in Configuration class is not used any more - and I pushed another commit removing it.